src/opm/common/utility/TimeService.cpp

```diff
@@ -225,4 +225,8 @@ Opm::TimeStampUTC Opm::operator+(const Opm::TimeStampUTC& lhs, std::chrono::dura
     return Opm::TimeStampUTC( Opm::TimeService::advance(Opm::asTimeT(lhs) , delta.count()) );
 }
 
+Opm::time_point Opm::asTimePoint(const Opm::TimeStampUTC& ts)
+{
+    return Opm::TimeService::from_time_t( Opm::asTimeT(ts) );
```

Comment:
If we're being pedantic, this function is already in the `Opm` namespace so should probably be implemented as
```C++
return TimeService::from_time_t(asTimeT(ts));
```
instead.

Reply:
Well - don't ask my why; but the main bulk of implementation of the time service is outside the opm namespace - I am just following the pattern in the file. Happy to change it though.

Reply:
> the main bulk of implementation of the time service is outside the opm namespace

I'm not sure I understand what you mean.  The `TimeService` namespace is nested within `Opm` and all `TimeService` functions are named
```
Opm::TimeService::something
```
so they're clearly within the `Opm` namespace.